easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-foss-2022b.eb

```diff
@@ -0,0 +1,57 @@
+# authors: Kenneth Hoste (Ghent University), George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# foss-2016b modified by:
+# Adam Huffman
+# The Francis Crick Institute
+# Changed the toolchain and dependency versions and 
+# also did not use java version since 11 should be standard
+# added PostgreSQL as a dependency since we no longer are installed postgresql at system level 
+# and postgres libraries are needed to function properly.
+# Updated by: Thomas Eylenbosch (Gluo N.V.)
+#
+
+easyblock = 'ConfigureMake'
+
+name = 'EMBOSS'
+version = '6.6.0'
+
+homepage = 'http://emboss.sourceforge.net/'
+description = """EMBOSS is 'The European Molecular Biology Open Software Suite'.
+ EMBOSS is a free Open Source software analysis package specially developed for
+ the needs of the molecular biology (e.g. EMBnet) user community."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = [
+    'ftp://emboss.open-bio.org/pub/EMBOSS/',
+    'ftp://emboss.open-bio.org/pub/EMBOSS/old/%(version_major_minor)s.0',
+]
+sources = [SOURCE_TAR_GZ]
+patches = ['EMBOSS_disable-embossupdate.patch']
```

Comment:
please use `EMBOSS-6.6.0_disable-embossupdate.patch` instead (with the version in the name)

Reply:
@eylenth any thoughts on the suggestion from Sam?